src/persist-client/examples/admin.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Commands for read-write administration of persist state
+
+use mz_build_info::BuildInfo;
+use mz_ore::metrics::MetricsRegistry;
+use mz_ore::now::SYSTEM_TIME;
+use mz_persist_client::{PersistConfig, ShardId};
+
+use prometheus::proto::{MetricFamily, MetricType};
+use std::str::FromStr;
+use tracing::info;
+
+use crate::inspect::StateArgs;
+
+/// Commands for read-write administration of persist state
+#[derive(Debug, clap::Args)]
+pub struct AdminArgs {
+    #[clap(subcommand)]
+    command: Command,
+}
+
+/// Individual subcommands of admin
+#[derive(Debug, clap::Subcommand)]
+pub(crate) enum Command {
+    /// Manually completes all fueled compactions in a shard.
+    ForceCompaction(StateArgs),
+}
+
+pub async fn run(command: AdminArgs) -> Result<(), anyhow::Error> {
+    match command.command {
+        Command::ForceCompaction(args) => {
+            let shard_id = ShardId::from_str(&args.shard_id).expect("invalid shard id");
+            // WIP hook this up to a real build_info
```

Comment:
@benesch any objections to putting persist-client in the ./bin/bump-version script?